styles/docs.css

```diff
@@ -295,3 +330,65 @@ input::placeholder{
 h1,h2{
     font-family: var(--brand-font);
 }
+
+h2{
+    color:var(--black1);
+}
+
+.active-tab .page-link{
+    color: var(--hot-pink);
+}
+
+.theme-switch-box {
+    display: flex;
+    align-items: center;
+  }
+
+  .theme-switch {
+    display: inline-block;
+    height: 34px;
+    position: relative;
+    width: 60px;
+  }
+  
+  .theme-switch input {
+    display:none;
+  }
+  
+  .slider {
+    background-color: var(--background-gray);
+    bottom: 0;
+    cursor: pointer;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    transition: .4s;
+  }
+  
+  .slider:before {
+    background-color: var(--hot-pink);
+    bottom: 4px;
+    content: "";
+    height: 26px;
+    left: 4px;
+    position: absolute;
+    transition: .4s;
+    width: 26px;
+  }
+  
+  input:checked + .slider {
```

Comment:
don't use input it's a very general selector and this style will apply to all of the input elements in your code. Instead use a specific class or id to target it.